src/main/webapp/css/main.css

```diff
@@ -25,8 +27,18 @@ nav ul {
   padding-left: 0;
 }
 
+ul {
```

Comment:
I'd suggest that you write a quick line comment saying what "ul" will be used for. Just so we don't have to try to guess given only the name of the variable.

You can find an example on line 17 in this main.css file.